README.rst

```diff
@@ -19,3 +19,15 @@ To use it, add ``sphinxcontrib.jquery`` as a Sphinx extension:
        "sphinxcontrib.jquery",
    ]
    ...
+
+Configuration
+-------------
+
+```
```

Comment:
This is reST, not markdown

Reply:
GitHub Flavored reST :upside_down_face: 

![image](https://user-images.githubusercontent.com/374612/217657050-b52a9afd-a0e6-4d89-8a79-654f52e91c08.png)

Reply:
oh yeah, been working too much on GH lately

Reply:
fixed!